src/nxdoc/web-ui/web-ui-release-notes.md

```diff
@@ -22,16 +22,17 @@ Starting from Web UI version `3.0.31` released in March 2024, functional tests w
 
 ## Recently Released Changes
 
-{{{multiexcerpt 'web-ui-updates' page='web-ui-release-notes-3-0-34'}}}
+{{{multiexcerpt 'web-ui-updates' page='web-ui-release-notes-3-0-35'}}}
 
 ---
 
 ## Previous Release Notes
 
-<!-- | [Web UI 3.1.10]({{page page='web-ui-release-notes-3-0-34'}})                  |  Saved searches can be restored and executed when using vocabularies with multiple levels, ellipsis and tooltips prevent long labels from being cut off in selection        | -->
+<!-- | [Web UI 3.0.35]({{page page='web-ui-release-notes-3-0-35'}})                  | WEB UI Improvement and bugfix release     | -->
```

Comment:
```suggestion
<!-- | [Web UI 3.0.35]({{page page='web-ui-release-notes-3-0-35'}})                  | Display the document creation button on all mobile devices     | -->
```
When talking about improvements, we need to be more specific and provide at least one highlight.